stripe/util.py

```diff
@@ -143,7 +185,10 @@ def convert_to_stripe_object(
     if isinstance(resp, list):
         return [
             convert_to_stripe_object(
-                i, api_key, stripe_version, stripe_account
+                cast("StripeObject", i),
```

Comment:
Why are we casting to StripeObject? Isn't StripeObject the output of convert_to_stripe_object instead of the input?

Reply:
Good catch, this was supposed to be "StripeResponse". At first glance I was mildly concerned that `StripeObject` was typechecking, but then I recalled that `StripeObject` extends from `Dict[str, Any]` which is one of the branches of `Resp`.